### ISSUE TITLE
ScrollArea's content leaking outside the ScrollArea's rect

### ISSUE THREAD

**Issue by Bowarc** [NemuiSen/ggegui#12]
The `Window Config` label, `Vertial Sync (reboot required)` label, the `Movement` button `+ another item below` are in a group, inside a `ScrollArea`. (i used `.show` on the `ScrollArea` if that can help 🤔)

![image](https://user-images.githubusercontent.com/63136904/221433430-bf97f14e-8489-472e-b1f0-d1b61706734f.png)

I don't know a lot on this subject so i asked in [Rust Game Dev discord](https://discord.gg/game-development-in-rust-676678179678715904)
[This message](https://discord.com/channels/676678179678715904/676685797524766720/1079457844941832293)

Is it a bug ? and if not, how can i fix it ?

**Comment by NemuiSen** [comment]
Can you give me a small example that replicates your problem?

**Comment by Bowarc** [comment]
> Can you give me a small example that replicates your problem?

Paste this as the `ggez::event::EventHandler::update` function:
```rust
fn update(&mut self, ctx: &mut Context) -> GameResult {
    let gui_ctx = self.gui.ctx();

    const WINDOW_SIZE: (f32, f32) = (300., 130.);

    egui::Window::new("UI")
        .fixed_size(WINDOW_SIZE)
        .show(&gui_ctx, |ui| {
            let text_style = egui::TextStyle::Small;
            let row_height = ui.text_style_height(&text_style);

            egui::ScrollArea::vertical()
                .stick_to_bottom(false)
                .max_height(WINDOW_SIZE.1)
                .max_width(WINDOW_SIZE.0)
                .min_scrolled_height(row_height)
                .show(ui, |ui| {
                    ui.group(|ui| {
                        ui.label("Config");
                        ui.group(|ui| {
                            ui.label("User config");
                            ui.vertical(|ui| {
                                ui.label("Dash direction method");
                                if ui.button("Movement").clicked() {
                                    // modify config
                                }
                            });
                        });
                        ui.group(|ui| {
                            ui.label("Window config");
                            ui.vertical(|ui| {
                                ui.label("Vertical sync (reboot required)");
                                if ui.button("false").clicked() {}
                            });
                        })
                    });
                });
        });
    self.gui.update(ctx);
    Ok(())
}
```

**Comment by NemuiSen** [comment]
![imagen](https://user-images.githubusercontent.com/40149702/221752161-47615757-ed21-44ba-acd7-f5a8d12f830f.png)
fixed with 88486a14d2a4ec163f1e0fa1514f14b5fae48738

**Comment by NemuiSen** [comment]
now the new version will be released un just a moment
